### PR TITLE
Remove Support menu option in Docs 

### LINKED ISSUE #3289: Remove Support option from top menu on Docs page
Because we now have a real Support page, we no longer need the one on Docs. We do need to retain the content under the page, however. For now we will move all of the content under a new section called Troubleshooting.

Process:

In a single PR:
1. Create a new Troubleshooting directory under /guides/database/
2. Move the index.md file FROM Support TO the new Troubleshooting dir ([howto](https://developercommunity.visualstudio.com/t/git-moving-a-file-performs-a-delete-and-add-not-a/34608))
3. Move the 5 Help Articles to be under the new Troubleshooting dir
4. Move the creating-bug-reports.mdx file to be under the new Troubleshooting dir

On a separate PR:
1. Create redirects for everything above.

Resources:
* [How to modify Docs home page](https://www.notion.so/prismaio/Doc-site-documentation-1c208d7fcb7a4290a8abe6a40f08b109)

### PR DESCRIPTION
**Pull request by tanberry**
Removing the Support menu option from the Docs top menu.

Fixes https://github.com/prisma/docs/issues/3289

### CODE REVIEW

**Review by janpio — CHANGES_REQUESTED** [review]
Generally looks good, most of the renames were recognized by GitHub.

Because of the additional text changes, the ones for the index pages were _not_, but that should not be a big deal.

What is missing are the necessary redirects. Those are needed before this can be merged so the old URLs keep working.

Also, for me the new location and URL is unexpected. You just started to only put database specific (databases and providers) into the "database guides" folders, and this is now something completely different again.

(Ah, just now noticed that this is still marked as draft - but I got a review request already. Hope this was intended.)

**Comment by tanberry** [comment]
Yes, thanks for your review @janpio , very much intended. And Yes, I know I needed a redirect, just hadn't gotten to it yet. That redirect PR is now created and ready... I'd like to merge them both asap... Idid see your comment about the location... those changes you mentioned will be part of the upcoming Info Architecture work; this is a quick fix to remove the Support page.

File: content/300-guides/050-database/880-troubleshooting-orm/index.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: 'Troubleshooting'
+metaTitle: 'Troubleshooting'
+metaDescription: 'Learn common troubleshooting tips, how to raise an issue or open a PR. '
+hidePage: false
+---
+
+## Need help?
+
+Refer to our [Support site](https://www.prisma.io/support/) for full information about getting help.
```
Comment (janpio) [inline content/300-guides/050-database/880-troubleshooting-orm/index.mdx]: No trailing slash on this URL:
```suggestion
Refer to our [Support site](https://www.prisma.io/support) for full information about getting help.
```

**Review by janpio — APPROVED** [review]
Actually unreviewed as redirects PR separate.

**Pull request merged**